pom.xml

```diff
@@ -668,19 +668,4 @@
             </build>
         </profile>
     </profiles>
-
-    <repositories>
```

Comment:
can we add commit messages for why to remove it?

Reply:
> can we add commit messages for why to remove it?

Update: modify it in title